packages/design-system/src/styles/components/_Tooltip.scss

```diff
@@ -17,8 +17,9 @@
   text-decoration-style: dotted;
   text-underline-position: under;
 }
+
 // Chrome needs this offset because it handles text-underline-position differently and without this the underline runs into descending letters like g,j,q,p,y.
-@media all and (-webkit-min-device-pixel-ratio: 0) and (min-resolution: 0.001dpcm) {
+@media (resolution >= 0.001dpcm) {
```

Comment:
I don't know why this is targeting webkit when it's a Chrome override

Reply:
This is smelly to me and deserves a deeper look with design. I'm not sure what the intention is because browsers have `text-decoration-skip-ink` to control how the underline flows with font descenders. And why is Chrome singled out? Just smelly...

Reply:
Chrome used to use webkit, but is that that old? It can't be from that time because I looked it up, and the switch from WebKit happened in 2013 I think.

Curious if the VRTs will detect any change on this.

Reply:
VRTs show no change, but I really wonder if we even need this. Even though they didn't change, the snapshots for the inline trigger in Safari and Firefox look kind of weird. The dotted lines are way too dense, and the line is really far away from the text. It actually looks like in these old snapshots that this media query was applying to those browsers even though I don't see it happening when I try it in those browsers myself. With your change, the query applies to those browsers, but those snapshots were taken when it wasn't supposed to.

Reply:
It's a hardware query and not a browser query though, right? If the hardware supports a resolution over that .001dpcm amount (which is a fraction of a dpi) it kicks in. So I don't know if VRT would capture this style change because... what is the dpi of a docker container? Or when it runs a job in Jenkins? Does it use our hardware to determine this stuff? 

That's why this is so darn smelly. It leads me to think this dotted style needs work. It's too brittle. 

If we tackle the Tooltip work, maybe this can be part of that?

Reply:
I think until we get to the bottom of this, we shouldn't necessarily expand it beyond webkit. This is something to take to our designers.

Reply:
Should I make a ticket for this? Or add it to our coworking agenda?